### PR TITLE
[Tests-Only]Add acceptance tests for operations on files with commas in the filename

### PR DESCRIPTION
**Pull request by jasson99**
## Description
This PR adds tests for operations on files with commas in the filename

## Related Issue
- #37664 

## How Has This Been Tested?
- https://github.com/owncloud/ocis/pull/374
- https://github.com/owncloud/ocis-reva/pull/370

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Review by dpakach — COMMENTED** [review]
lgtm :+1:

**Review by individual-it — DISMISSED** [review]
for more places to add files/folders with commas search for tests using
`strän`

**Review by individual-it — COMMENTED** [review]
looks good
please also make a test-run on ocis-reva

File: tests/acceptance/features/apiTrashbin/trashbinRestore.feature [hunk 1]
```diff
@@ -116,6 +116,7 @@ Feature: Restore deleted files/folders
       | old      | /textfile0.txt          | PARENT/textfile0.txt |
       | new      | /textfile0.txt          | PARENT/textfile0.txt |
 
+
```
Comment (phil-davis) [inline tests/acceptance/features/apiTrashbin/trashbinRestore.feature:119]: Bonus line. If you need to change or rebase the PR, then remove this line.

File: tests/acceptance/features/apiWebdavOperations/downloadFile.feature [hunk 2]
```diff
@@ -129,3 +129,17 @@ Feature: download file
...
       | old         |
       | new         |
+
+    Scenario Outline: Download a file with comma in the filename
+      Given using <dav_version> DAV path
+      And user "Alice" has uploaded file with content "file with comma in filename" to <filename>
+      When user "Alice" downloads file <filename> using the WebDAV API
+      Then the downloaded content should be "file with comma in filename"
+      Examples:
+        | dav_version | filename      |
```
Comment (phil-davis) [inline tests/acceptance/features/apiWebdavOperations/downloadFile.feature:139]:
```suggestion
        | dav_version | filename       |
```
If you need to rebase or change this PR, then lineup the column

File: tests/acceptance/features/webUICreateDelete/deleteFilesFolders.feature [hunk 3]
```diff
@@ -13,16 +13,23 @@ Feature: deleting files and folders
...
+    And user "Alice" has created folder "folder,with,comma"
+    And the user has browsed to the files page
     When the user deletes the following elements using the webUI
       | name                                  |
       | simple-folder                         |
       | lorem.txt                             |
       | strängé नेपाली folder                 |
       | strängé filename (duplicate #2 &).txt |
+      | s,a,m,p,l,e.txt                       |
+      | folder,with,comma                      |
```
Comment (phil-davis) [inline tests/acceptance/features/webUICreateDelete/deleteFilesFolders.feature:26]:
```suggestion
      | s,a,m,p,l,e.txt                       |
      | folder,with,comma                     |
```
column not aligned

File: tests/acceptance/features/webUIFavorites/favoritesFile.feature [hunk 4]
```diff
@@ -10,12 +10,16 @@ Feature: Mark file as favorite
     Given user "Alice" has been created with default attributes and without skeleton files
     And user "Alice" has uploaded file "filesForUpload/data.zip" to "/data.zip"
     And user "Alice" has uploaded file "filesForUpload/lorem.txt" to "/lorem.txt"
+    And user "Alice" has uploaded file with content "file with comma" to "s,a,m,p,l,e.txt"
     And user "Alice" has logged in using the webUI
     And the user has browsed to the files page
     When the user marks file "data.zip" as favorite using the webUI
+    And the user marks file "s,a,m,p,l,e.txt" as favorite using the webUI
     Then file "data.zip" should be marked as favorite on the webUI
+    Then file "s,a,m,p,l,e.txt" should be marked as favorite on the webUI
```
Comment (phil-davis) [inline tests/acceptance/features/webUIFavorites/favoritesFile.feature:19]:
```suggestion
    And file "s,a,m,p,l,e.txt" should be marked as favorite on the webUI
```

File: tests/acceptance/features/webUIFavorites/favoritesFile.feature [hunk 5]
```diff
@@ -10,12 +10,16 @@ Feature: Mark file as favorite
...
     When the user reloads the current page of the webUI
     Then file "data.zip" should be marked as favorite on the webUI
+    Then file "s,a,m,p,l,e.txt" should be marked as favorite on the webUI
```
Comment (phil-davis) [inline tests/acceptance/features/webUIFavorites/favoritesFile.feature:22]:
```suggestion
    And file "s,a,m,p,l,e.txt" should be marked as favorite on the webUI
```

**Review by phil-davis — COMMENTED** [review]
Looks good, just minor comments.

Please adjust, rebase and squash the commits. Then run a test PR from ocis-reva. (ocis-reva is currently up-to-date with the tests in core, so it should pass)

**Comment by phil-davis** [comment]
drone CI got annoying errors: https://drone.owncloud.com/owncloud/core/25877/129/7
`Error: No such image: docker.io/owncloudci/php:7.4`

when you rebase and squash we will get another CI  run anyway.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**